run.sh

```diff
@@ -1,5 +1,12 @@
-#!/usr/bin/env bash
+#!/usr/env/bin bash
```

Comment:
This line is causing `bad interpreter: /usr/env/bin: no such file or directory` for me

Reply:
Reverting it works

Reply:
thank you!